spec/factories/book_suggestion.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :book_suggestion do
+    title     { Faker::Book.title }
+    author    { Faker::Book.author }
+    link      { Faker::Internet.url }
+    publisher { Faker::Book.publisher }
+    year      { Faker::Number.between(2016, 2018) }
+    price     { Faker::Number.between(300, 1500) }
+    synopsis  { Faker::Movie.quote }
+    user_id   { nil }
```

Comment:
User traits for this, so you can easily create a BookSuggestion with and without a user_id. It's kind of redundant to create a trait for only one value but i would like you to learn about them 😉